### PR TITLE
Add flag to provide custom components directory path for dapr run

### PR DESCRIPTION
**Pull request by ksashikumar**
# Description

This PR adds `--components-path` flag to `dapr run` command

## Issue reference

Please reference the issue this PR will close: https://github.com/dapr/cli/issues/321

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Extended the documentation

---
This change is [Reviewable](https://reviewable.io/reviews/dapr/cli/326)

### CODE REVIEW

File: cmd/run.go [hunk 1]
```diff
@@ -243,6 +245,7 @@ func init() {
 	RunCmd.Flags().StringVarP(&logLevel, "log-level", "", "info", "Sets the log verbosity. Valid values are: debug, info, warn, error, fatal, or panic. Default is info")
 	RunCmd.Flags().IntVarP(&maxConcurrency, "max-concurrency", "", -1, "controls the concurrency level of the app. Default is unlimited")
 	RunCmd.Flags().StringVarP(&protocol, "protocol", "", "http", "tells Dapr to use HTTP or gRPC to talk to the app. Default is http")
+	RunCmd.Flags().StringVarP(&componentsPath, "components-path", "", "", "Path for components directory.")
```
Comment (LMWF) [inline cmd/run.go:248]: It would be good to indicate the default here.  Eg append "Default is ./components"

Comment (ksashikumar) [reply cmd/run.go]: Makes sense! I've made the change 👍 Thanks for the suggestion 🙂

Comment (LMWF) [reply cmd/run.go]: I actually just meant as a text comment in the 5th arg.  ie what you had but
```
RunCmd.Flags().StringVarP(&componentsPath, "components-path", "", "", "Path for components directory.  Default is ./components.")
```
I think putting a default value with a slash in the 4th arg may not work on some OSes.

Comment (ksashikumar) [reply cmd/run.go]: My bad! 🤦 Made the change now 👍 Thanks 🙂

**Comment by LMWF** [comment]
Thanks for your contribution!

**Pull request merged**